tests/data/xdmf/basement3/3HumpsTest/3_humps_mesh.2dm

```diff
@@ -1,4 +1,5 @@
 MESH2D   #created automatically via meshModel tool
+NUM_MATERIALS_PER_ELEM 2
```

Comment:
The test source files cannot be unfortunately modified. The test files comes from the runs of the solvers and they are not usually made by hand. So modifying them means that the we are not testing the real world example.

Reply:
Ah, of course - I'll revert them to their previous versions and add support for face elevation without having the tag set, then.